docs/reference/contrib/table_block.rst

```diff
@@ -99,13 +100,15 @@ When defining a TableBlock, Wagtail provides the ability to pass an optional ``t
 Configuration Options
 ^^^^^^^^^^^^^^^^^^^^^
 
-Every key in the ``table_options`` dictionary maps to a `handsontable <https://handsontable.com/>`_ option. These settings can be changed to alter the behaviour of tables in Wagtail. The following options are available:
+The ``table_header_choice`` option determines if rows or columns are treated as headings. Valid values are ``row``, ``column``, ``both``, and ``neither``. The default is ``row``. This will cause the elements in the first row of your table to be rendered as ``th`` elements with ``scope = row``.
+
+The rest of the keys in the ``table_options`` dictionary each map to a `handsontable <https://handsontable.com/>`_ option. These settings can be changed to alter the behaviour of tables in Wagtail. The following options are available:
 
 * `minSpareRows <https://handsontable.com/docs/6.2.2/Options.html#minSpareRows>`_ - The number of rows to append to the end of an empty grid. The default setting is 0.
 * `startRows <https://handsontable.com/docs/6.2.2/Options.html#startRows>`_ - The default number of rows for a new table.
 * `startCols <https://handsontable.com/docs/6.2.2/Options.html#startCols>`_ - The default number of columns for new tables.
-* `colHeaders <https://handsontable.com/docs/6.2.2/Options.html#colHeaders>`_ - Can be set to ``True`` or ``False``. This setting designates if new tables should be created with column headers. **Note:** this only sets the behaviour for newly created tables. Page editors can override this by checking the the “Column header” checkbox in the table editor in the Wagtail admin.
```

Comment:
Should we keep that note or is it no longer relevant? I’m not entirely sure I understand it but would be keen to hear more.

Reply:
Removing all reference to colHeaders and rowHeaders might be the best option. After all, we do not document every Handsontable configuration option; we only include the ones most relevant to our use of the library.

Reply:
Indeed, that good to me.

Reply:
I have removed colHeaders and rowHeaders from our docs. Mentioning them was confusing.